web/src/main/java/se/inera/intyg/rehabstod/web/controller/api/dto/PopulateFiltersResponseDTO.java

```diff
@@ -20,20 +20,22 @@
 package se.inera.intyg.rehabstod.web.controller.api.dto;
 
 import java.util.List;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import se.inera.intyg.rehabstod.service.diagnos.dto.DiagnosKapitel;
 import se.inera.intyg.rehabstod.web.model.Lakare;
 
+@Data
+@NoArgsConstructor
 public class PopulateFiltersResponseDTO {
     private List<Lakare> activeDoctors;
+    private List<DiagnosKapitel> allDiagnosisChapters;
+    private List<DiagnosKapitel> enabledDiagnosisChapters;
 
-    public PopulateFiltersResponseDTO(List<Lakare> activeDoctors) {
-        this.activeDoctors = activeDoctors;
-    }
-
-    public List<Lakare> getActiveDoctors() {
-        return activeDoctors;
-    }
-
-    public void setActiveDoctors(List<Lakare> activeDoctors) {
+    public PopulateFiltersResponseDTO(List<Lakare> activeDoctors, List<DiagnosKapitel> allDiagnosisChapters,
```

Comment:
Since we decided to use @NoArgsConstructor, maybe we can use @AllArgsConstructor to generate this constructor?